modules/aws/master/master.tf

```diff
@@ -97,7 +97,7 @@ resource "aws_security_group" "master" {
   ingress {
     from_port   = 0
     to_port     = 0
-    protocol    = 94
+    protocol    = 4
```

Comment:
what does this mean? any link to docs or something like that? or is it just some magic value only AWS knows ?

Reply:
here we go https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml